code/modules/mob/living/living_defines.dm

```diff
@@ -71,3 +71,5 @@
 
 	var/blood_volume = 0 //how much blood the mob has
 	var/obj/effect/proc_holder/ranged_ability //Any ranged ability the mob has, as a click override
+
+	var/slipping = FALSE
```

Comment:
is there not some value set when the user is already slipping? surely there is already a flag.